### ISSUE TITLE
accuracy UNET

### ISSUE THREAD

**Issue by luantunez** [sandialabs/bcnn#6]
Hello and thank you for sharing your work.
I am having trouble since the metrics of my BCNN with custom 2D images are not improving at all. 
Can you think of a reason?
I am using 
Tensorflow 2.4.1
Tensorflow-probability 0.12.1

I have tried several models all across the web, and I am having the same problem. The data shouldn´t be the problem, since a regular CNN trains perfectly.

**Comment by tmlabonte** [comment]
Performance is not guaranteed in TensorFlow 2, since a lot changed from TensorFlow 1 to 2 and I'm not sure how the changes interact with TensorFlow Probability. As a general tip, you may want to adjust the learning rate or KL annealing schedule to fit your dataset.
